src/include/distributed/commands.h

```diff
@@ -398,7 +401,7 @@ typedef enum CascadeOperationType
 	INVALID_OPERATION = 1 << 0,
 
 	/* execute UndistributeTable on each relation */
-	UNDISTRIBUTE_TABLE = 1 << 1,
+	CASCADE_FKEY_UNDISTRIBUTE_TABLE = 1 << 1,
```

Comment:
seems that we should have either this enum or your defines (like `#define UNDISTRIBUTE_TABLE`), but not both